rotkehlchen/chain/ethereum/modules/l2/loopring.py

```diff
@@ -256,7 +256,7 @@ def _api_query(
                     # This two codes can appear if there is an error with the
                     # key as tested here
                     # https://github.com/rotki/rotki/pull/2569#issuecomment-800151821
-                    if code and code in (100001, 101002):
```

Comment:
So from the docs this error code is returned for invalid api key.

If that is the case you will need to edit the comment also and edit the exception. It's no longer an APIKeyMismatch but an invalid api key. Also adjust error messages where it's caught.